docs/claimantmodel/Book.md

```diff
@@ -0,0 +1,130 @@
+# Claimant Model In-Depth
+
+## Introduction
+
+The Claimant Model is very concisely introduced in the [Core Model](./CoreModel.md).
+While this concise documentation works well as a refresher for those that already understand the model, it has
+empirically proved to be a tough introduction to those new to the domain.
+This longer form text on the Claimant Model serves as a more gentle on-ramp to understanding this powerful tool.
+
+This text is broken into sections, but is intended to be read from start to finish.
+
+## Motivation
+
+Prior to the Claimant Model being developed, new transparency projects were designed by copying and modifying
+patterns from existing deployments, most notably [Certificate Transparency](https://certificate.transparency.dev/) (CT).
+While this allows those familiar with CT to reach consensus on a rough design quickly, it has a number of drawbacks:
+
+1. Those that don't already understand CT now have to read [RFC 6962](https://www.rfc-editor.org/rfc/rfc6962) to understand
+   the domain that the other designers are drawing from
+1. There are still terms in CT that people disagree on the precise definition of (e.g. Auditor and Monitor)
+1. CT uses mechanisms such as SCTs that transparency systems should aim to avoid unless strictly necessary
```

Comment:
Would it be worthwhile to provide a brief explanation for this comment, given this is one of the primary designs we want to avoid copying over from CT?

Reply:
I agree we should have public documentation about why these promises are a non-trivial change to the security model and what this entails, but I will take a strong default position that this is not an appropriate location for this content to live. We should link to it from here when it exists though. Do either of you have a counter argument to this position?

Specifically I'd want to be convinced that this documentation which is about the claimant model would be improved by adding content about a) the history of CT, and/or b) design considerations of SCTs.

Reply:
I think it's fine for documentation about CT and SCTs to be linked from elsewhere, but I don't know if we do have those docs yet. Given that we really do want to discourage the use of SCTs, I think we either need to create a brief doc about the downsides of them and alternatives, or briefly summarize the issue here.